assets/css/main.scss

```diff
@@ -70,6 +70,70 @@ $bh_brown: #a65a36 !default;
     }
 }
 
+.page__hero--overlay .page__lead {
```

Comment:
can you comment any changes you made so I know why

Reply:
I don't know where this came from 🤔 
I did do an upgrade of some packages to get around an issue where liquid wouldn't work with jekyll on the version of ruby i was running, and the version of ruby that was suggested didn't work on Apple silicone. I wonder if this got updated as a part of new package versions?